src/main/java/org/datadog/jenkins/plugins/datadog/DatadogGlobalConfiguration.java

```diff
@@ -140,6 +141,9 @@ public class DatadogGlobalConfiguration extends GlobalConfiguration {
     private Secret targetApiKey = null;
     private String targetCredentialsApiKey = null;
     private Secret usedApiKey = null;
+    private Secret targetApplicationKey = null;
+    private String targetCredentialsApplicationKey = null;
```

Comment:
Is there a reason we want to keep this a `String` rather than a `Secret`?

Reply:
This isn't the key itself, but an identifier of the `Credentials` that store the key.